docs/cli/dnx-migration.md

```diff
@@ -0,0 +1,153 @@
+project file are pretty much the same, with mild differences based on the scenarios. Let's see what those differences are. 
+
+If you are building a console application, you need to add the following snippet to your project file:
+
+```json
+"compilationOptions": {
+    "emitEntryPoint": true
+}
+```
+
+This instructs `dotnet build` to emit an entry point for your application, effectively making your code runnable. If 
+you are building a class library, simply omit the above section. Of course, once you add the above snippet to your 
+`project.json` file, you need to add a static entry point; with the move off DNX, the DI services it provided are no 
+longer available and thus this needs to be a basic .NET entry point: `static void Main()`.
+
+If you have a "commands" section in your `project.json`, you can remove it. Some of the commands that used to exist as 
+DNU commands, such as Entity Framework CLI commands, are being ported to be 
+per-project extensions to the CLI. If you built your own commands that you are using in your projects, you need to 
+replace them with CLI extensions. In this case, the `commands` node in `project.json` needs to be replaced by the 
+`tools` node and it needs to list the tools dependencies as explained in the 
+[CLI extensibility section](overview.md#extensibility). 
+
+After these things are done, you need to decide which type of portability you wish for you app. With .NET Core, we have 
+invested into providing a spectrum of portability options that you can choose from. For instance, you may want to have 
+a fully *portable* application or you may want to have a *self-contained* application. Portable application option is more 
+like .NET Framework applications work: it needs a shared component to execute it on the target machine 
+(the .NET Core runtime). The self-contained application doesn't require a shared runtime, but you have to produce one application for each OS you wish to support. 
+These portability types and more are discussed in the [application portability type](app-types.md) document. 
+
+Once you make a call on what type of portability you want, you need to change your targeted framework(s). If you were 
+writing applications for .NET Core, you were most likely using `dnxcore50` as  your targeted framework. With the CLI 
+and the changes that the new [.NET Standard Library](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md) 
+brought, the framework needs to be one of the following:
+
+1. `netcoreapp1.0` - if you are writing applications on .NET Core (including ASP.NET Core applications)
+2. `netstandard1.5` - if you are writing class libraries for .NET Core
+
+Your `project.json` is now mostly ready. You need to go through your dependencies list and update the dependencies to 
+their newer versions, especially if you are using ASP.NET Core dependencies. If you used to use separate packages for BCL APIs, 
```

Comment:
@DrewScoggins there will be a separate asp.net core rc1 -> rc2 doc written/created by the ASP.NET team. I don't really know enough to make that quality content. 

/cc @danroth27